hironx_ros_bridge/scripts/hironx.py

```diff
@@ -79,7 +79,9 @@
     try:
         ros = ROS_Client()
     except socket.error as e:
-        print("\033[31m%s\033[0m" % (e.strerror))
+        errormsg = 'No ROS Master found. Without it, you cannot use ROS from this script, but can use RTM. ' + \
```

Comment:
"No ROS Master found" situation raises [ROSInitException](https://github.com/start-jsk/rtmros_hironx/blob/f523a54e9cd96af54a29d5613730b91600eb95a1/hironx_ros_bridge/src/hironx_ros_bridge/ros_client.py#L86), not socket.error.